azkaban-execserver/src/package/conf/azkaban.properties

```diff
@@ -25,6 +25,10 @@ executor.flow.threads=30
 jetty.connector.stats=true
 executor.connector.stats=true
 
+# External analyzer settings
```

Comment:
Could you add more comment? e.g. how the %s works?
Comment out these configs. Users can uncomment and customize.